epub33/core/index.html

```diff
@@ -11777,6 +11785,12 @@ <h3>Substantive changes since <a href="https://www.w3.org/TR/2022/CR-epub-33-202
 						Recommendation</a></h3>
 
 				<ul>
+					<li>11-Jan-2023: Marked the <code>dir</code> attribute under-implemented due to lack of support in
+						reading systems. See <a href="https://github.com/w3c/epub-specs/pull/2515">pull request
+						2515</a>.</li>
+					<li>09-Jan-2023: Marked the <code>rtl</code> value of the <code>dir</code> attribute
+						under-implemented due to lack of support in reading systems. See <a
+							href="https://github.com/w3c/epub-specs/pull/2515">pull request 2515</a>.</li>
```

Comment:
I presume this entry should be removed from the list of changes; the entry of 11-Jan-2023 supersedes it.

Reply:
Ya, not sure what happened there. I modified the existing entry so I must have missed the duplication when resolving the merge conflict. Gone now.